.claude/commands/shared/model_utils.sh

```
@@ -253,18 +253,18 @@ zen_mcp_consensus() {
+    # mcp__pal__tiered_consensus --prompt "$request (Topic: $topic)" --level <1|2|3> --findings "..."
 
-    echo "✨ Zen MCP layered consensus available for: $request (Topic: $topic)"
+    echo "✨ PAL tiered consensus available for: $request (Topic: $topic)"
     echo "🎯 Organization: $org_level, Models: $model_count, Cost: $cost_threshold"
-    echo "💡 Call mcp__zen__layered_consensus directly from your slash commands"
+    echo "💡 Call mcp__pal__tiered_consensus directly from your slash commands"
     return 0
 
     # Final fallback to sequential calls
-    echo "🔄 Sequential consensus (zen tools not available)"
+    echo "🔄 Sequential consensus (pal tools not available)"
```

Shellcheck (0.11.0)

[info] Command appears to be unreachable. Check usage (or ignore if invoked indirectly).

(SC2317)